Exercise_9/src/MyFramework/customTags/CustomLink.js

```diff
@@ -0,0 +1,19 @@
+const customlink = {
+  template: '<a></a>',
+  beforeMount: (app, element, componentData) => {
+    for (let index = 0; index < componentData.length; index += 1) {
+      const attributeName = componentData[index].name;
+      const attributeValue = componentData[index].value;
+
+      if (attributeName === 'href') {
```

Comment:
This is hard-code. It would be better to have it as an attribute. Something like this:
```javascript
onclick: (app, href) => app.navigate(href)
```